modules/custom-status/custom-status.php

```diff
@@ -30,10 +30,16 @@ class Custom_Status extends Module {
 	private $custom_statuses_cache = [];
 
 	// This is taxonomy name used to store all our custom statuses
-	const TAXONOMY_KEY = 'post_status';
+	const TAXONOMY_KEY = 'vw_post_status';
 
 	const SETTINGS_SLUG = 'vw-custom-status';
 
+	// The metadata keys for the custom status term
+	const METADATA_POSITION_KEY = 'position';
+	const METADATA_REQ_EDITORIAL_FIELDS_KEY = 'required_metadata_fields';
```

Comment:
I went back and forth on calling this ID or fields. I left it as fields to mark the purpose of this field. Unlike the users field, there are no full EM objects being given back.

Reply:
How about `required_metadata_field_ids`?

Reply:
I had that, but it became such a mouthful that I reverted it. I'll change the fields to ids